apps/ledger-live-desktop/tests/page/swap.page.ts

```diff
@@ -115,10 +116,7 @@ export class SwapPage extends AppPage {
   }
 
   @step("Select exchange quote $0 with rate $1")
-  async selectExchangeQuote(
-    providerName: "changelly" | "cic" | "oneinch" | "paraswap",
```

Comment:
[Comment] I don't know if it's normal but it you look at the providerinside `Swap.ts` file there is only 2 provider (changelly and oneinch).  So `CIC`and `Paraswap` providers are missing.

Reply:
atm those are the only ones I use in my tests scenarios, there are more to come in another PR with `EXODUS` for instance
nonetheless, previously you were limited to 4 values, which isn't the case anymore
also won't use providers that require registration or use their own webview for the time being